### ISSUE TITLE
Speech Recognition/Whisper word level scores or confidence output

### ISSUE THREAD

**Issue by wobbble** [xenova/whisper-web#32]
Hey,
Big thanks for awesome project!

It possible to add score/confidence for word level output when using Speech Recognition/Whisper model?
Would appreciate any direction/comments or suggestion where to dig to add it. 
Happy to submit PR if I will success in it.

Thanks!

**Comment by decoder-sh-david** [comment]
Seconded, I have also not been able to successfully get word-level timestamps while running on webgpu. Would love to have both!

**Comment by hlevring** [comment]
Looks like it has been supported for a while
https://github.com/huggingface/transformers.js/releases/tag/2.4.0

**Comment by xenova** [comment]
Indeed, Transformers.js v3 w/ word-level timestamps should work! @decoder-sh-david let me know if you run into any issues.
